ui/components/Layout.tsx

```diff
@@ -141,12 +144,26 @@ function Layout({ className, children }: Props) {
                 value={getParentNavValue(currentPage)}
               >
                 {_.map(navItems, (n) => (
-                  <StyleLinkTab
-                    key={n.value}
-                    label={n.label}
-                    to={formatURL(n.value)}
-                    value={n.value}
-                  />
+                  <Flex column>
+                    <StyleLinkTab
+                      key={n.value}
+                      label={n.label}
+                      to={formatURL(n.value)}
+                      value={n.value}
+                    />
+                    {_.map(n.subItems, (sub) => {
+                      return (
+                        <StyleLinkTab
```

Comment:
Looking at Figma, sub items should be `12px` below the main one. This looks a like a little bigger gap.